test-cases/tests/installation/a30-validate-installation-of-rhoam-addon-and-integration-with-ld.md

```diff
@@ -57,7 +59,7 @@ Host prefix: /26
 3. Fill in the following parameters and click on `Install`
 
 ```
-CIDR range: "10.1.0.0/24"
+CIDR range: "10.1.0.0/24" (note this down to use it later for another verification step)
```

Comment:
Since this was written we've decided to use a /26 as default. It doesn't really change the verification but if it doesn't delay things could we update that also here?

Reply:
sure, will do